src/lib/rules/player.hh

```diff
@@ -33,38 +35,40 @@ struct Player
     std::string name;
 };
 
-using Player_sptr = std::shared_ptr<Player>;
-
-struct Players final : utils::IBufferizable
+class Players final : public utils::IBufferizable
 {
+    using PlayersVector = std::vector<std::shared_ptr<Player>>;
+
+public:
     Players() = default;
-    Players(std::vector<Player_sptr> players) : players(players) {}
+    Players(PlayersVector players) : players_{players} {}
 
     void handle_buffer(utils::Buffer& buf) override
     {
         if (buf.serialize())
         {
-            for (auto& p : players)
+            for (const auto& p : players_)
```

Comment:
The usual naming nit, but maybe this was already done in master.

Reply:
Fixed.